drivers/dac/ad9144/ad9144.c

```diff
@@ -687,7 +687,6 @@ int32_t ad9144_short_pattern_test(struct ad9144_dev *dev,
 
 	uint32_t dac = 0;
 	uint32_t sample = 0;
-	uint8_t status = 0;
```

Comment:
please put in the whole word in the title "remove" instead of "rem"

no need to write the whole path drivers:dac:ad9144:ad9144.c
ad9144.c is enough, because this file is unique and resides only in one place in no-OS anyway